rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] :person, :book should be `attr_accessor` because it should be modified.

Reply:
done

Reply:
- [ ] You still need to address this because `:person` and `: book` is still `attr_reader`